packages/calcite-components/src/components/table/table.scss

```diff
@@ -15,20 +15,20 @@
 }
 
 :host {
-  @apply table;
+  @apply flex;
```

Comment:
This would be a breaking change, right? Are we allowing a breaking change here?

Reply:
Yeah, not sure if there's another way to fix this without this change. The resultant Chromatic changes are minimal (https://www.chromatic.com/build?appId=6266d45509d7eb004aa254fb&number=3263) - maybe @jcfranco can make a call?

Reply:
Sounds good.

We had discussed changes like this being breaking changes since users apps may be expecting the component to display a certain way. Maybe we can get away with these changes if we note them more prominently.

Reply:
Marking as draft for now.

Reply:
I think we can proceed with this w/o it being breaking for the following reasons:
 
* `display: table` ➡️  `display: flex` still has a block flow parent-wise (flex-specific styling might kick in, but not likely due to its current display and any overrides to use flex would still be honored)
* Slotted children of `calcite-table` are all table-related, so the styles of these should be encapsulated already, and user content would be unaffected